docs/docs/cmd/tenant/people/people-profilecardproperty-list.mdx

```diff
@@ -53,13 +52,8 @@ m365 tenant people profilecardproperty list
           ]
         }
       ]
-    },
-    {
-      "directoryPropertyName": "Alias",
```

Comment:
I would leave that to show that the custom properties look differently then others

Reply:
We made an agreement that we always show one object in the result preview so it doesn't get too big. If the result actually differs, we usually create an extra section where we show the possible outcomes.
However in this case, according to me there is no structural change in the result. It's just an empty array. If we have to define all outcomes that could have empty arrays, we'd have a lot of work.
Consider the command `outlook message get` for example. CC BCC recipients is also an array of recipient objects. In this case we also show just one object instead of multiple.

Reply:
I understand, but in this case I see benefit in pointing it out, and still it does not take a lot of place don't you agree?

Reply:
It doesn't take much space here no, but to be honest, I think we have to keep being consistent. There's not a single command that prints more than one object in the response section. If we want to show differences in output we usually introduce multiple subsections as you can see here: https://pnp.github.io/cli-microsoft365/cmd/planner/task/task-add#response

However, we only do that when the structure of the output changes. For the example above, the 2nd output has extra properties (`description`, `references`, `checklist`) and in this command, the structure doesn't change. Some arrays can be empty indeed, but that is inherent to an array.
If we show every possible output for every command, the response section can get gigantic for some commands (it's true that the output here is not that huge, but I think we should be consistent).

Therefore, generally speaking, I think we should always show the output as rich as possible (try to provide a value for every property in the output), and then the user can reason for himself how the output would look for him.

Reply:
ok sure